tools/monitoring/main.go

```diff
@@ -78,20 +72,22 @@ func main() {
 		log.Fatal(err)
 	}
 
-	ctx := context.Background()
-	client, err = subscriber.NewSubscriberClient(ctx, projectID, subName)
-	if err != nil {
-		log.Fatalf("Failed to initialize the subscriber %+v", err)
-	}
-
 	err = gcs.Authenticate(context.Background(), *serviceAccount)
 	if err != nil {
 		log.Fatalf("Failed to authenticate gcs %+v", err)
 	}
 
-	wfClient = alert.Setup(client, db, &alert.MailConfig{Config: mailConfig, Recipients: alertEmailRecipients})
+	alertClient, err = alert.Setup(db, mailConfig)
+	if err != nil {
+		log.Fatalf("Failed to setup test-infra monitoring: %v\n", err)
+	}
+	alertClient.RunAlerting()
 
-	wfClient.RunAlerting()
+	imageClient, err := valimage.Setup(mailConfig)
```

Comment:
This should not be part of the infra monitoring package. I would prefet this to be its own tool re-using the packages defined in monitoring.